### PR TITLE
Set up GitHub actions

### PR DESCRIPTION
**Pull request by peterstuart**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install ALSA
+        run: sudo apt-get install libasound2-dev
```
Comment (helixbass) [inline .github/workflows/main.yml:14]: This looks unnecessarily copied over from `pedals`?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version: 2
```
Comment (helixbass) [inline .github/dependabot.yml:1]: This looks nicely declarative, what does it do?

**Review by helixbass — APPROVED** [review]
@peterstuart thanks for setting this up, see comments

File: .github/workflows/main.yml (hunk 1 above)
Comment (peterstuart) [reply .github/workflows/main.yml:14]: I tried without it, but wmidi seems to require it on linux. Maybe alsa provides midi as well?

File: .github/dependabot.yml (hunk 2 above)
Comment (peterstuart) [reply .github/dependabot.yml:1]: I think version 2 is the "native to github" version of Dependabot, rather than the only semi-third party thing.

**Pull request merged**